target/linux/kirkwood/config-5.4

```diff
@@ -53,9 +53,7 @@ CONFIG_ARM_L1_CACHE_SHIFT=5
 CONFIG_ARM_PATCH_PHYS_VIRT=y
 # CONFIG_ARM_THUMB is not set
 CONFIG_ARM_UNWIND=y
-CONFIG_ATA=y
 CONFIG_ATAGS=y
-CONFIG_ATA_LEDS=y
```

Comment:
This patch is the only thing that still bugs me.

It doesn't sound really right to me to have CONFIG_ATA_LEDS=y built into the kernel but CONFIG_ATA to be built as module.

Despite, we still have CONFIG_ARCH_WANT_LIBATA_LEDS=y here, which creates the "unmet direct dependencies" warning.

I'd rather say let's keep CONFIG_ATA, CONFIG_ATA_LEDS and CONFIG_SATA_PMP in the config, and only use the module for CONFIG_SATA_MV, where it's straightforward.

Reply:
I think that's also reasonable if CONFIG_ATA does not take so much space.